referenceframe/frame.go

```diff
@@ -267,9 +270,11 @@ func (sf *staticFrame) Geometries(input []Input) (*GeometriesInFrame, error) {
 	if len(input) != 0 {
 		return nil, NewIncorrectInputLengthError(len(input), 0)
 	}
-	m := make(map[string]spatial.Geometry)
-	m[sf.Name()] = sf.geometry.Transform(spatial.NewZeroPose())
-	return NewGeometriesInFrame(sf.name, m), nil
+	newGeom := sf.geometry.Transform(spatial.NewZeroPose())
+	if newGeom.Label() == "" {
```

Comment:
Theres a pattern across these changes of transforming the geometry and then setting the Label.  I don't see why we couldn't just return the labeled geometry from the transform function, since the transformed geometry will have access to its own name.  Will clean up the code a good deal I think

Reply:
We do return the label of the geometry if it has one. This is specifically for the case where the geometry of a frame is itself unlabeled